### PR TITLE
Add SBOM documentation

### PR DESCRIPTION
**Pull request by Chr1st1anSears**
This PR adds two new files, one for CLI and one for the web UI, that tell users how to download SBOMs for their components.

### CODE REVIEW

File: docs/modules/ROOT/pages/cli/cli-sbom.adoc [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+----
+
++
+Example:
+
++
+[source]
+----
+$ cosign download sbom quay.io/redhat-appstudio/user-workload@sha256:<output omitted>
+----
```
Comment (psturc) [inline docs/modules/ROOT/pages/cli/cli-sbom.adoc:77]: would it be worth to add a note that cosign might print out this warning after running the command and state if user should (not) worry about it?
```
❯ cosign download sbom quay.io/psturc/test-images@sha256:51f321b27c2c0b3669b03fea4c889575cd72b66f4a741b7cdeaf21439f8fb5df > /tmp/sbom.txt
WARNING: Downloading SBOMs this way does not ensure its authenticity. If you want to ensure a tamper-proof SBOM, download it using 'cosign download attestation <image uri>' or verify its signature using 'cosign verify --key <key path> --attachment sbom <image uri>'.
Found SBOM of media type: application/vnd.cyclonedx+json

❯ cosign version
...
GitVersion:    v2.0.0
```

File: docs/modules/ROOT/pages/how-to-guides/webui-sbom.adoc [hunk 2]
```diff
@@ -0,0 +1,108 @@
+= Download SBOMs
+
+:toc: left
+:icons: font
+:numbered:
+:source-highlighter: highlightjs
+
+== Software Bill of Materials
+A software bill of materials (SBOM) provides greater transparency for your software supply chain. In {ProductName}, an SBOM lists all the software libraries that a xref:glossary/index.adoc#_component[component] uses. Those libraries may enable specific functionality, facilitate development, or serve some other purpose.
```
Comment (psturc) [inline docs/modules/ROOT/pages/how-to-guides/webui-sbom.adoc:9]:
```suggestion
A software bill of materials (SBOM) provides greater transparency for your software supply chain. In {ProductName}, an SBOM lists all the software libraries that a xref:../glossary/index.adoc#_component[component] uses. Those libraries may enable specific functionality, facilitate development, or serve some other purpose. 
```

File: docs/modules/ROOT/pages/how-to-guides/webui-sbom.adoc [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+
+== Downloading an SBOM
+
+Currently, {ProductName} requires you to download the SBOM by running a CLI command. This means you need to access {ProductName} through your CLI, and you must install an additional CLI tool. We apologize for this inconvenience. However, the {ProductName} web UI still accelerates the process of downloading an SBOM, and in the future, you will be able to download the SBOM directly through the web UI. 
+
+*Prerequisites*
+
+* Login to the {ProductName} web UI
+
+* Login to {ProductName} xref:cli/getting_started_in_cli.adoc[through the CLI].
```
Comment (psturc) [inline docs/modules/ROOT/pages/how-to-guides/webui-sbom.adoc:21]:
```suggestion
* Login to {ProductName} xref:../cli/getting_started_in_cli.adoc[through the CLI].
```

**Review by psturc — COMMENTED** [review]
please review my suggestions about updates of cross reference links. Also I had just one minor question, otherwise LGTM!

File: docs/modules/ROOT/pages/cli/cli-sbom.adoc [hunk 4]
```diff
@@ -0,0 +1,133 @@
+= Download SBOMs
+
+:toc: left
+:icons: font
+:numbered:
+:source-highlighter: highlightjs
+
+== Software Bill of Materials
+A software bill of materials (SBOM) provides greater transparency for your software supply chain. In {ProductName}, an SBOM lists all the software libraries that a xref:glossary/index.adoc#_component[component] uses. Those libraries may enable specific functionality, facilitate development, or serve some other purpose.
```
[the remark this answers is not in the document]
Comment (Chr1st1anSears) [reply docs/modules/ROOT/pages/cli/cli-sbom.adoc:9]: Made this change :+1:

File: docs/modules/ROOT/pages/how-to-guides/webui-sbom.adoc (hunk 3 above)
Comment (Chr1st1anSears) [reply docs/modules/ROOT/pages/how-to-guides/webui-sbom.adoc:21]: Also made this change

File: docs/modules/ROOT/pages/cli/cli-sbom.adoc [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+
+== Downloading an SBOM
+
+*Prerequisites*
+
+* Install the link:https://docs.sigstore.dev/cosign/installation/[Cosign] CLI tool.
+
+* Install the link:https://stedolan.github.io/jq/download/[jq] CLI tool.
+
+* xref:cli/getting_started_in_cli.adoc[Login] to {ProductName}.
```
[the remark this answers is not in the document]
Comment (Chr1st1anSears) [reply docs/modules/ROOT/pages/cli/cli-sbom.adoc:21]: When I tried making this change, it actually caused the xref to fail for some reason, so I'm keeping this as is.

File: docs/modules/ROOT/pages/cli/cli-sbom.adoc (hunk 1 above)
Comment (Chr1st1anSears) [reply docs/modules/ROOT/pages/cli/cli-sbom.adoc:77]: Great suggestion--I added a note about that.

**Review by psturc — APPROVED** [review]

**Review by jseseCCS — APPROVED** [review]
approved for merge per new RHTAP content review process.

**Pull request merged**